models/reviews.js

```diff
@@ -36,3 +37,42 @@ exports.fetchReviews = () => {
       return reviews;
     });
 };
+
+exports.addReviewComment = (review_id, comment) => {
+  const allowedKeys = ["username", "body"];
+  for (const key of Object.keys(comment)) {
+    if (!allowedKeys.includes(key)) {
+      return Promise.reject({
+        status: 400,
+        msg: "Incorrect properties on comment object.",
```

Comment:
If my memory serves, you include this message, but don't test for it; 
- it'd be nice to see your excellent messages in the test file, too